### PR TITLE
AWS management Script 

### PR DESCRIPTION
**Pull request by shantamsultania**
hey @kaustubhgupta this is for - issue #856

Fixes: #856
I am making one issue of the same, not 2 different issues also please recheck the level for the script as I think to know it's able to solve 2 issues with the same so it can be a bit level 3 material (just a suggestion )

### CODE REVIEW

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- Create a folder named: "AWS Management Scripts" and put both the folders under it.
- Create a separate readme for this folder with relevant screenshots: AWS Automation Script for AWS endorsement management

**Comment by kaustubhgupta** [comment]
Also, fix these errors:

image

**Review by Sanskrutim17 — APPROVED** [review]

**Comment by shantamsultania** [comment]
> Also, fix these errors:
> 
> image

hey @kaustubhgupta these changes are just call variables as currently this script is not being used by anyone this error is being displayed that is of unused variables I don't think there is a change required for the same

**Comment by shantamsultania** [comment]
> * Create a folder named: "AWS Management Scripts" and put both the folders under it.
> * Create a separate readme for this folder with relevant screenshots: AWS Automation Script for AWS endorsement management

ok will do the same

**Comment by kaustubhgupta** [comment]
> > Also, fix these errors:
> > image
> 
> hey @kaustubhgupta these changes are just call variables as currently this script is not being used by anyone this error is being displayed that is of unused variables I don't think there is a change required for the same

Okay got it, that's the problem with the linter, it can't handle dynamic modules.

**Comment by shantamsultania** [comment]
hey @kaustubhgupta I have made the requested changes please look into it as for screenshots/images in AWS Automation Script for AWS endorsement that will be not required as I think they are management scripts not a particular scripts related to a endorsement and this can change from user/ client point of view

File: AWS Management Scripts/AWS Automation Script for AWS endorsement management/Readme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Aws Script for AWS Management
+
+This python script can be used to manage a AWS endorsement.
+
+this provides features like :
```
Comment (kaustubhgupta) [inline AWS Management Scripts/AWS Automation Script for AWS endorsement management/Readme.md:5]:
```suggestion
This provides features like :
```

File: AWS Management Scripts/AWS Automation Script for AWS endorsement management/Readme.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+EC2 :
```
Comment (kaustubhgupta) [inline AWS Management Scripts/AWS Automation Script for AWS endorsement management/Readme.md:7]:
```suggestion
### EC2 :
```

File: AWS Management Scripts/AWS Automation Script for AWS endorsement management/Readme.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+1. Create your own Key based SnapShots 
+2. Delete SnapShots 
+3. Delete/ Terminate any EC2 instance which does not have a user/ any specific tag 
+4. stop any useless Running Ec2 instance
+
+RDS :
```
Comment (kaustubhgupta) [inline AWS Management Scripts/AWS Automation Script for AWS endorsement management/Readme.md:14]:
```suggestion
### RDS : 
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by antrikshmisri** [comment]
@kaustubhgupta I don't have much knowledge about AWS so, can't comment on this.

**Comment by shantamsultania** [comment]
hey @kaustubhgupta I think all the minor changes requested must be cleared out and we can merge it up if the PA or the mentors don't have any issues with it

**Comment by santushtisharma10** [comment]
@shantamsultania is it a flask app?

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@shantamsultania remove this empty [file](https://github.com/shantamsultania/Amazing-Python-Scripts/blob/master/AWS%20Management%20Scripts/AWS%20testing%20Script/__init__.py)

**Comment by shantamsultania** [comment]
hey @santushtisharma10 there are 2 scripts in this 1 AWS testing script is a Flask app but not 2. that is AWS script for endorsement

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Comment by shantamsultania** [comment]
hey @kaustubhgupta as all the reviews are approved can we can merge this so that I can start working on the next issue.

**Comment by kaustubhgupta** [comment]
> hey @kaustubhgupta as all the reviews are approved can we can merge this so that I can start working on the next issue.

A review from one mentor is still missing. You can work on other issues by making different branches for each issue but as you have made the PR using the master branch, you will need to wait for this PR to be merged.

**Comment by shantamsultania** [comment]
> > hey @kaustubhgupta as all the reviews are approved can we can merge this so that I can start working on the next issue.
> 
> A review from one mentor is still missing. You can work on other issues by making different branches for each issue but as you have made the PR using the master branch, you will need to wait for this PR to be merged.

I think you are waiting for a comment by @antrikshmisri he has already commented above that he is not that much aware of AWS thus he cannot comment on the same I think we can move this up for PA review and if he is ok we can close this up

**Comment by kaustubhgupta** [comment]
> > > hey @kaustubhgupta as all the reviews are approved can we can merge this so that I can start working on the next issue.
> > 
> > 
> > A review from one mentor is still missing. You can work on other issues by making different branches for each issue but as you have made the PR using the master branch, you will need to wait for this PR to be merged.
> 
> I think you are waiting for a comment by @antrikshmisri he has already commented above that he is not that much aware of AWS thus he cannot comment on the same I think we can move this up for PA review and if he is ok we can close this up

We have another mentor @Kushal997-das

**Review by antrikshmisri — APPROVED** [review]

**Comment by shantamsultania** [comment]
> > > > hey @kaustubhgupta as all the reviews are approved can we can merge this so that I can start working on the next issue.
> > > 
> > > 
> > > A review from one mentor is still missing. You can work on other issues by making different branches for each issue but as you have made the PR using the master branch, you will need to wait for this PR to be merged.
> > 
> > 
> > I think you are waiting for a comment by @antrikshmisri he has already commented above that he is not that much aware of AWS thus he cannot comment on the same I think we can move this up for PA review and if he is ok we can close this up
> 
> We have another mentor @Kushal997-das

ok waiting for his review on the same

**Review by avinashkranjan — APPROVED** [review]
LGTM..Great Work @shantamsultania

**Pull request merged**